app/helpers/home_helper.rb

```diff
@@ -6,4 +6,20 @@ def home
       home_path
     end
   end
+
+  def home_page?
+    current_page? home
+  end
+
+  def about_page?
+    current_page? about_path
+  end
+
+  def contact_page?
+    current_page? contact_path
+  end
+
+  def home_pages?
```

Comment:
Could we rename this method to `header_links?` or even `header_pages?`? Asking because these page links sit under the header partial and `home_pages?` can be confused with `home_page?`

Reply:
Yeah, I totally agree with you! I read your comment before, and I was like "what a hell, if home page or not home page... oh, ok, there's an 's' there..."
Good catch!